### PR TITLE
Feat/windows

### LINKED ISSUE #55: Windows Setup
Add to root README.md all packages/programs/etc to install for new Shift3 Windows users.

### PR DESCRIPTION
**Pull request by stephengtuggy**
## Changes
1. Root `README.md`: Adds instructions for setting up apps on Windows, post-install
2. Adds a batch file to automate setting up most of the apps
3. `server-side/os/README.md`: Fills out this file with links to Ubuntu and Windows Server setup instructions
4. Adds Windows Server setup README

## Purpose
Add setup instructions for both Windows dev environments and production Windows servers

## Approach
Puts the Windows dev instructions in the root readme, as specified in issue #55 . Puts the Windows Server instructions under `server-side/os/`.

Closes #55

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,9 +54,23 @@
...
+- From the same PowerShell or Command Prompt window, run the command `shutup10`. Here, you can easily configure a number of settings to have Windows respect your privacy a little more.
+- After running shutup10, restart Windows if necessary.
+- Install Android Studio.
+- Install the regular version of Visual Studio (as opposed to Visual Studio Code). Visual Studio Community Edition may be sufficient, or you may need a paid license for Visual Studio Professional, Enterprise, or similar. Either way, ask your PM (Project Manager).
+- Ask your Project Manager if you need to install anything else.
+
+Using Chocolatey has several advantages:
+1. You can install most of the software you need just by running a single script.
+2. You can update all of this software later, by running `choco upgrade all`, as Administrator.
+3. Chocolatey will automatically skip installing browser toolbars and junk like that.
```
Comment (carlosvargas) [inline README.md]: I feel that we should also drop a line about the Windows Subsystem for Linux (https://docs.microsoft.com/en-us/windows/wsl/faq).

Comment (stephengtuggy) [reply README.md]: > I feel that we should also drop a line about the Windows Subsystem for Linux (https://docs.microsoft.com/en-us/windows/wsl/faq).

@carlosvargas Sounds good. When I have a chance, I will add that in.

**Comment by ryekerjh** [comment]
@stephengtuggy can you add a link to this in the [Getting Started docs](https://github.com/Shift3/standards-and-practices#new-to-shift3) for new hires? There's a blank `Windows` section that needs filling...

File: README.md (hunk 1 above)
Comment (stephengtuggy) [reply README.md]: Done!

**Comment by stephengtuggy** [comment]
> @stephengtuggy can you add a link to this in the [Getting Started docs](https://github.com/Shift3/standards-and-practices#windows-install) for new hires? There's a blank `Windows` section that needs filling...

@ryekerjh I actually did put these instructions in the `Windows` section of the Getting Started docs. I put them in the wrong place at first, but then I corrected myself. That's all included in this PR.

Let me know if I need to make any further changes.

**Pull request merged**